lib/mixduty/client.ex

```diff
@@ -20,4 +20,19 @@ defmodule Mixduty.Client do
       ]
     }
   end
+
+  @doc """
+  Create the client for Bearer authorization token
+  #### Example
+      Mixduty.Client.new(:bearer, "bearerauthtoken")
+  """
+  def new(:bearer, auth) do
```

Comment:
I don't like the type as the 1st argument because you can't pipe `token |> Client.new()`.

```suggestion
  def new(auth, [type: :bearer]) do
```